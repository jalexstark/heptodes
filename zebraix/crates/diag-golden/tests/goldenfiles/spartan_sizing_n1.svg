<?xml version="1.0" encoding="UTF-8"?>
<svg xmlns="http://www.w3.org/2000/svg" xmlns:xlink="http://www.w3.org/1999/xlink" width="400pt" height="300pt" viewBox="0 0 400 300">
<defs>
<g>
<g id="glyph-0-0">
<path d="M 0.375 -2.140625 L 0.375 -2.875 L 2.640625 -2.875 L 2.640625 -2.140625 Z M 0.375 -2.140625 "/>
</g>
<g id="glyph-0-1">
<path d="M 5.15625 -1.515625 L 4.1875 -1.515625 L 4.1875 0 L 3.390625 0 L 3.390625 -1.515625 L 0.203125 -1.515625 L 0.203125 -2.21875 L 3.328125 -6.703125 L 4.1875 -6.703125 L 4.1875 -2.25 L 5.15625 -2.25 Z M 3.390625 -4.359375 C 3.390625 -4.679688 3.394531 -4.953125 3.40625 -5.171875 C 3.414062 -5.398438 3.425781 -5.617188 3.4375 -5.828125 L 3.40625 -5.828125 C 3.351562 -5.710938 3.289062 -5.582031 3.21875 -5.4375 C 3.144531 -5.300781 3.070312 -5.191406 3 -5.109375 L 1 -2.25 L 3.390625 -2.25 Z M 3.390625 -4.359375 "/>
</g>
<g id="glyph-0-2">
<path d="M 0.671875 -0.5 C 0.671875 -0.726562 0.726562 -0.890625 0.84375 -0.984375 C 0.957031 -1.078125 1.09375 -1.125 1.25 -1.125 C 1.40625 -1.125 1.539062 -1.078125 1.65625 -0.984375 C 1.769531 -0.890625 1.828125 -0.726562 1.828125 -0.5 C 1.828125 -0.28125 1.769531 -0.117188 1.65625 -0.015625 C 1.539062 0.078125 1.40625 0.125 1.25 0.125 C 1.09375 0.125 0.957031 0.078125 0.84375 -0.015625 C 0.726562 -0.117188 0.671875 -0.28125 0.671875 -0.5 Z M 0.671875 -0.5 "/>
</g>
<g id="glyph-0-3">
<path d="M 4.890625 -3.34375 C 4.890625 -2.625 4.8125 -2.007812 4.65625 -1.5 C 4.507812 -0.988281 4.273438 -0.59375 3.953125 -0.3125 C 3.628906 -0.0390625 3.195312 0.09375 2.65625 0.09375 C 1.90625 0.09375 1.347656 -0.207031 0.984375 -0.8125 C 0.628906 -1.425781 0.453125 -2.269531 0.453125 -3.34375 C 0.453125 -4.0625 0.523438 -4.675781 0.671875 -5.1875 C 0.816406 -5.707031 1.050781 -6.101562 1.375 -6.375 C 1.707031 -6.644531 2.132812 -6.78125 2.65625 -6.78125 C 3.40625 -6.78125 3.960938 -6.476562 4.328125 -5.875 C 4.703125 -5.269531 4.890625 -4.425781 4.890625 -3.34375 Z M 1.28125 -3.34375 C 1.28125 -2.425781 1.382812 -1.738281 1.59375 -1.28125 C 1.800781 -0.832031 2.15625 -0.609375 2.65625 -0.609375 C 3.15625 -0.609375 3.515625 -0.832031 3.734375 -1.28125 C 3.953125 -1.738281 4.0625 -2.425781 4.0625 -3.34375 C 4.0625 -4.25 3.953125 -4.929688 3.734375 -5.390625 C 3.515625 -5.847656 3.15625 -6.078125 2.65625 -6.078125 C 2.15625 -6.078125 1.800781 -5.847656 1.59375 -5.390625 C 1.382812 -4.929688 1.28125 -4.25 1.28125 -3.34375 Z M 1.28125 -3.34375 "/>
</g>
<g id="glyph-0-4">
<path d="M 3.3125 0 L 2.515625 0 L 2.515625 -4.65625 C 2.515625 -4.925781 2.515625 -5.140625 2.515625 -5.296875 C 2.523438 -5.460938 2.535156 -5.632812 2.546875 -5.8125 C 2.453125 -5.707031 2.363281 -5.617188 2.28125 -5.546875 C 2.195312 -5.484375 2.09375 -5.398438 1.96875 -5.296875 L 1.265625 -4.71875 L 0.828125 -5.265625 L 2.640625 -6.671875 L 3.3125 -6.671875 Z M 3.3125 0 "/>
</g>
<g id="glyph-0-5">
<path d="M 4.859375 0 L 0.453125 0 L 0.453125 -0.6875 L 2.203125 -2.453125 C 2.535156 -2.785156 2.816406 -3.082031 3.046875 -3.34375 C 3.273438 -3.601562 3.445312 -3.859375 3.5625 -4.109375 C 3.6875 -4.367188 3.75 -4.644531 3.75 -4.9375 C 3.75 -5.3125 3.640625 -5.59375 3.421875 -5.78125 C 3.203125 -5.96875 2.914062 -6.0625 2.5625 -6.0625 C 2.238281 -6.0625 1.953125 -6.003906 1.703125 -5.890625 C 1.460938 -5.785156 1.21875 -5.628906 0.96875 -5.421875 L 0.515625 -5.984375 C 0.785156 -6.203125 1.085938 -6.382812 1.421875 -6.53125 C 1.765625 -6.6875 2.144531 -6.765625 2.5625 -6.765625 C 3.1875 -6.765625 3.679688 -6.609375 4.046875 -6.296875 C 4.410156 -5.984375 4.59375 -5.546875 4.59375 -4.984375 C 4.59375 -4.640625 4.519531 -4.3125 4.375 -4 C 4.226562 -3.695312 4.023438 -3.394531 3.765625 -3.09375 C 3.515625 -2.800781 3.21875 -2.488281 2.875 -2.15625 L 1.484375 -0.78125 L 1.484375 -0.75 L 4.859375 -0.75 Z M 4.859375 0 "/>
</g>
</g>
</defs>
<path fill="none" stroke-width="1" stroke-linecap="butt" stroke-linejoin="miter" stroke="rgb(0%, 0%, 0%)" stroke-opacity="1" stroke-miterlimit="10" d="M 18.183594 286.363281 L 18.183594 13.636719 L 381.816406 13.636719 L 381.816406 286.363281 Z M 18.183594 286.363281 "/>
<path fill="none" stroke-width="0.45" stroke-linecap="butt" stroke-linejoin="miter" stroke="rgb(0%, 0%, 0%)" stroke-opacity="1" stroke-dasharray="4.5 3.15" stroke-miterlimit="10" d="M 18.183594 150 L 381.816406 150 M 18.183594 204.546875 L 381.816406 204.546875 M 18.183594 259.089844 L 381.816406 259.089844 M 18.183594 95.453125 L 381.816406 95.453125 M 18.183594 40.910156 L 381.816406 40.910156 "/>
<path fill="none" stroke-width="0.45" stroke-linecap="butt" stroke-linejoin="miter" stroke="rgb(0%, 0%, 0%)" stroke-opacity="1" stroke-dasharray="4.5 3.15" stroke-miterlimit="10" d="M 290.910156 286.363281 L 290.910156 13.636719 M 230.304688 286.363281 L 230.304688 13.636719 M 169.695312 286.363281 L 169.695312 13.636719 M 109.089844 286.363281 L 109.089844 13.636719 M 48.484375 286.363281 L 48.484375 13.636719 M 351.515625 286.363281 L 351.515625 13.636719 "/>
<g fill="rgb(0%, 0%, 0%)" fill-opacity="1">
<use xlink:href="#glyph-0-0" x="40.484375" y="298.229492"/>
<use xlink:href="#glyph-0-1" x="43.484375" y="298.229492"/>
<use xlink:href="#glyph-0-2" x="48.484375" y="298.229492"/>
<use xlink:href="#glyph-0-3" x="51.484375" y="298.229492"/>
</g>
<g fill="rgb(0%, 0%, 0%)" fill-opacity="1">
<use xlink:href="#glyph-0-3" x="284.410156" y="298.229492"/>
<use xlink:href="#glyph-0-2" x="289.410156" y="298.229492"/>
<use xlink:href="#glyph-0-3" x="292.410156" y="298.229492"/>
</g>
<g fill="rgb(0%, 0%, 0%)" fill-opacity="1">
<use xlink:href="#glyph-0-4" x="345.015625" y="298.229492"/>
<use xlink:href="#glyph-0-2" x="350.015625" y="298.229492"/>
<use xlink:href="#glyph-0-3" x="353.015625" y="298.229492"/>
</g>
<g fill="rgb(0%, 0%, 0%)" fill-opacity="1">
<use xlink:href="#glyph-0-0" x="-2.492188" y="261.866211"/>
<use xlink:href="#glyph-0-5" x="0.507812" y="261.866211"/>
<use xlink:href="#glyph-0-2" x="5.507812" y="261.866211"/>
<use xlink:href="#glyph-0-3" x="8.507812" y="261.866211"/>
</g>
<g fill="rgb(0%, 0%, 0%)" fill-opacity="1">
<use xlink:href="#glyph-0-3" x="0.507812" y="152.776367"/>
<use xlink:href="#glyph-0-2" x="5.507812" y="152.776367"/>
<use xlink:href="#glyph-0-3" x="8.507812" y="152.776367"/>
</g>
<g fill="rgb(0%, 0%, 0%)" fill-opacity="1">
<use xlink:href="#glyph-0-5" x="0.507812" y="43.686523"/>
<use xlink:href="#glyph-0-2" x="5.507812" y="43.686523"/>
<use xlink:href="#glyph-0-3" x="8.507812" y="43.686523"/>
</g>
<path fill="none" stroke-width="1" stroke-linecap="butt" stroke-linejoin="miter" stroke="rgb(0%, 0%, 65%)" stroke-opacity="1" stroke-miterlimit="10" d="M 168.765625 83.882812 C 168.765625 87.617188 163.167969 87.617188 163.167969 83.882812 C 163.167969 80.152344 168.765625 80.152344 168.765625 83.882812 Z M 198.109375 74.832031 C 198.109375 78.566406 192.511719 78.566406 192.511719 74.832031 C 192.511719 71.097656 198.109375 71.097656 198.109375 74.832031 Z M 244.585938 73.21875 C 244.585938 76.953125 238.984375 76.953125 238.984375 73.21875 C 238.984375 69.488281 244.585938 69.488281 244.585938 73.21875 Z M 297.167969 104.546875 C 297.167969 108.277344 291.566406 108.277344 291.566406 104.546875 C 291.566406 100.8125 297.167969 100.8125 297.167969 104.546875 Z M 285.261719 184.269531 C 285.261719 188.003906 279.660156 188.003906 279.660156 184.269531 C 279.660156 180.535156 285.261719 180.535156 285.261719 184.269531 Z M 204.109375 236.125 C 204.109375 239.859375 198.511719 239.859375 198.511719 236.125 C 198.511719 232.390625 204.109375 232.390625 204.109375 236.125 Z M 144.25 239.054688 C 144.25 242.789062 138.648438 242.789062 138.648438 239.054688 C 138.648438 235.320312 144.25 235.320312 144.25 239.054688 Z M 115.148438 227.703125 C 115.148438 231.4375 109.550781 231.4375 109.550781 227.703125 C 109.550781 223.972656 115.148438 223.972656 115.148438 227.703125 Z M 101.550781 215.996094 C 101.550781 219.726562 95.953125 219.726562 95.953125 215.996094 C 95.953125 212.261719 101.550781 212.261719 101.550781 215.996094 Z M 94.921875 206.425781 C 94.921875 210.160156 89.324219 210.160156 89.324219 206.425781 C 89.324219 202.691406 94.921875 202.691406 94.921875 206.425781 Z M 91.554688 198.914062 C 91.554688 202.648438 85.953125 202.648438 85.953125 198.914062 C 85.953125 195.183594 91.554688 195.183594 91.554688 198.914062 Z M 89.808594 193 C 89.808594 196.734375 84.207031 196.734375 84.207031 193 C 84.207031 189.269531 89.808594 189.269531 89.808594 193 Z M 88.914062 188.277344 C 88.914062 192.011719 83.316406 192.011719 83.316406 188.277344 C 83.316406 184.542969 88.914062 184.542969 88.914062 188.277344 Z M 88.914062 188.277344 "/>
<path fill="none" stroke-width="1" stroke-linecap="butt" stroke-linejoin="miter" stroke="rgb(60%, 0%, 0%)" stroke-opacity="1" stroke-miterlimit="10" d="M 165.964844 83.882812 L 171.761719 81.605469 L 178.273438 79.339844 L 185.597656 77.152344 L 193.839844 75.140625 L 203.09375 73.445312 L 213.4375 72.261719 L 224.894531 71.867188 L 237.410156 72.628906 L 250.761719 75.023438 L 264.496094 79.617188 L 277.824219 87.03125 L 289.558594 97.804688 L 298.1875 112.1875 L 302.085938 129.910156 L 299.976562 150 L 291.433594 170.859375 L 277.148438 190.628906 L 258.753906 207.71875 L 238.296875 221.183594 L 217.65625 230.808594 L 198.1875 236.925781 L 180.671875 240.15625 L 165.414062 241.195312 L 152.402344 240.65625 L 141.449219 239.054688 L 132.304688 236.769531 L 124.703125 234.078125 L 118.382812 231.179688 L 113.136719 228.203125 L 108.765625 225.234375 L 105.121094 222.335938 L 102.070312 219.535156 L 99.511719 216.859375 L 97.355469 214.308594 L 95.539062 211.894531 L 94.003906 209.613281 L 92.699219 207.457031 L 91.589844 205.425781 L 90.648438 203.507812 L 89.84375 201.703125 L 89.15625 199.996094 L 88.570312 198.390625 L 88.066406 196.871094 L 87.640625 195.433594 L 87.273438 194.078125 L 86.960938 192.792969 L 86.695312 191.574219 L 86.46875 190.417969 L 86.277344 189.320312 L 86.113281 188.277344 "/>
</svg>
