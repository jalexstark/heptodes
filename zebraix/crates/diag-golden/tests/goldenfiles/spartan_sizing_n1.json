{
  "sizing_scheme": "Fill",
  "axes_range": [
    -4.5,
    -2.5,
    1.5,
    2.5
  ],
  "padding": [
    0.05
  ],
  "drawables": [
    {
      "drawable": {
        "Polyline": {
          "line_closure_choice": "Closed",
          "locations": [
            [
              -4.5,
              -2.5
            ],
            [
              -4.5,
              2.5
            ],
            [
              1.5,
              2.5
            ],
            [
              1.5,
              -2.5
            ]
          ]
        }
      }
    },
    {
      "drawable": {
        "Lines": {
          "line_choice": "Light",
          "start": [
            [
              -4.5,
              0.0
            ]
          ],
          "end": [
            [
              1.5,
              0.0
            ]
          ],
          "offsets": [
            [
              0.0,
              0.0
            ],
            [
              -0.0,
              -1.0
            ],
            [
              -0.0,
              -2.0
            ],
            [
              0.0,
              1.0
            ],
            [
              0.0,
              2.0
            ]
          ]
        }
      }
    },
    {
      "drawable": {
        "Lines": {
          "line_choice": "Light",
          "start": [
            [
              0.0,
              -2.5
            ]
          ],
          "end": [
            [
              0.0,
              2.5
            ]
          ],
          "offsets": [
            [
              0.0,
              0.0
            ],
            [
              -1.0,
              -0.0
            ],
            [
              -2.0,
              -0.0
            ],
            [
              -3.0,
              -0.0
            ],
            [
              -4.0,
              -0.0
            ],
            [
              1.0,
              0.0
            ]
          ]
        }
      }
    },
    {
      "drawable": {
        "Text": {
          "size_choice": "Small",
          "offset_choice": "Both",
          "anchor_vertical": "Top",
          "texts": [
            {
              "content": "-4.0",
              "location": [
                -4.0,
                -2.5
              ]
            },
            {
              "content": "0.0",
              "location": [
                0.0,
                -2.5
              ]
            },
            {
              "content": "1.0",
              "location": [
                1.0,
                -2.5
              ]
            }
          ]
        }
      }
    },
    {
      "drawable": {
        "Text": {
          "size_choice": "Small",
          "offset_choice": "Both",
          "anchor_horizontal": "Right",
          "texts": [
            {
              "content": "-2.0",
              "location": [
                -4.5,
                -2.0
              ]
            },
            {
              "content": "0.0",
              "location": [
                -4.5,
                0.0
              ]
            },
            {
              "content": "2.0",
              "location": [
                -4.5,
                2.0
              ]
            }
          ]
        }
      }
    },
    {
      "layer": 20,
      "drawable": {
        "Points": {
          "color_choice": "Blue",
          "centers": [
            [
              -2.061559595959596,
              1.2121212121212122
            ],
            [
              -1.5773659010600707,
              1.3780918727915197
            ],
            [
              -0.8105530791788851,
              1.4076246334310851
            ],
            [
              0.05704583333333361,
              0.8333333333333333
            ],
            [
              -0.13941151832460694,
              -0.6282722513089009
            ],
            [
              -1.4783725563909775,
              -1.5789473684210527
            ],
            [
              -2.4660693877551023,
              -1.63265306122449
            ],
            [
              -2.9462236033519553,
              -1.424581005586592
            ],
            [
              -3.1706009165902844,
              -1.2098991750687442
            ],
            [
              -3.2799649425287356,
              -1.0344827586206897
            ],
            [
              -3.3355416160672586,
              -0.8967772069126576
            ],
            [
              -3.364358700284091,
              -0.7883522727272727
            ],
            [
              -3.379098746867168,
              -0.7017543859649122
            ]
          ]
        }
      }
    },
    {
      "layer": 30,
      "drawable": {
        "Polyline": {
          "color_choice": "Red",
          "locations": [
            [
              -2.061559595959596,
              1.2121212121212122
            ],
            [
              -1.9659545902373485,
              1.2539184952978057
            ],
            [
              -1.8585054309915292,
              1.295465869456901
            ],
            [
              -1.7376134668892598,
              1.335559265442404
            ],
            [
              -1.60161547099189,
              1.372426699937617
            ],
            [
              -1.4489228070175435,
              1.4035087719298245
            ],
            [
              -1.2782799683293742,
              1.4251781472684086
            ],
            [
              -1.0892098478066248,
              1.432408236347359
            ],
            [
              -0.8827298885511649,
              1.4184397163120568
            ],
            [
              -0.6624153493699878,
              1.374570446735395
            ],
            [
              -0.43580645161290293,
              1.2903225806451613
            ],
            [
              -0.21592871572871514,
              1.1544011544011543
            ],
            [
              -0.022255821371610043,
              0.9569377990430619
            ],
            [
              0.12010849220104024,
              0.6932409012131715
            ],
            [
              0.18440589318600412,
              0.3683241252302021
            ],
            [
              0.1495904761904764,
              -4.2294210461910723e-16
            ],
            [
              0.008624091778202767,
              -0.382409177820268
            ],
            [
              -0.22707895716946053,
              -0.7448789571694604
            ],
            [
              -0.530547442680776,
              -1.0582010582010584
            ],
            [
              -0.8680694942903754,
              -1.3050570962479613
            ],
            [
              -1.2086814814814821,
              -1.4814814814814816
            ],
            [
              -1.5299078353253657,
              -1.5936254980079683
            ],
            [
              -1.818900118063755,
              -1.6528925619834711
            ],
            [
              -2.0706618599791016,
              -1.671891327063741
            ],
            [
              -2.285379132040628,
              -1.6620498614958448
            ],
            [
              -2.4660693877551023,
              -1.63265306122449
            ],
            [
              -2.6169425885755606,
              -1.5907447577729574
            ],
            [
              -2.7424247912652544,
              -1.5414258188824663
            ],
            [
              -2.8466550658271323,
              -1.4882655981682884
            ],
            [
              -2.933274244751664,
              -1.4336917562724014
            ],
            [
              -3.0053747126436785,
              -1.379310344827586
            ],
            [
              -3.0655294653957728,
              -1.326150020721094
            ],
            [
              -3.1158539182602176,
              -1.274840644919385
            ],
            [
              -3.1580760640108956,
              -1.225740551583248
            ],
            [
              -3.1936033509152963,
              -1.179025752404592
            ],
            [
              -3.223581560283688,
              -1.1347517730496455
            ],
            [
              -3.2489444704657826,
              -1.092896174863388
            ],
            [
              -3.2704547282738807,
              -1.0533875987550874
            ],
            [
              -3.288737000220897,
              -1.0161254694057875
            ],
            [
              -3.3043046392806046,
              -0.9809932556713672
            ],
            [
              -3.3175810426540284,
              -0.9478672985781991
            ],
            [
              -3.3289167283624184,
              -0.9166225982725187
            ],
            [
              -3.3386029899786434,
              -0.8871365204534253
            ],
            [
              -3.3468828295227864,
              -0.8592910848549945
            ],
            [
              -3.3539597300014363,
              -0.8329742926899325
            ],
            [
              -3.360004713804714,
              -0.8080808080808081
            ],
            [
              -3.3651620397317474,
              -0.7845122105529545
            ],
            [
              -3.369553816839347,
              -0.7621769679647492
            ],
            [
              -3.373283754350511,
              -0.7409902324014819
            ],
            [
              -3.376440220502491,
              -0.720873529099968
            ],
            [
              -3.379098746867168,
              -0.7017543859649122
            ]
          ]
        }
      }
    }
  ]
}
